R/paths.R

```diff
@@ -269,7 +269,7 @@ packratCacheVersion <- function() {
 }
 
 cacheLibDir <- function(...) {
-  file.path(appDataDir(), packratCacheVersion(), "library", ...)
+  file.path(appDataDir(), packratCacheVersion(), paste0(R.version$major, ".", R.version$minor), "library", ...)
```

Comment:
This will set a cache using the full R version (e.g. R 3.6.0), whereas by default R shares libraries across 'patch' versions of R (e.g. using just 3.6 in the path).

Do you think separate patch versions of R should get separate caches, or does it make sense to share them?

Reply:
Interesting that rsconnect has major minor specific caches. Ive only found the problem to be major version specific but think major minor would likely mean less need for manual edits to cache more if something did go wrong. That and aligning with rsconnect sounds sensible.